docs/src/man/guide.md

```diff
@@ -61,13 +61,15 @@ a single empty file at the moment, but we'll be adding to it later on.
 Add the following to your `make.jl` file
 
 ```julia
+push!(LOAD_PATH,"../src/")
```

Comment:
Could you point me to the line(s) in guide.md that you're referring to (which state that `Example` is a normal package)?